src/main/java/Level0.java

```diff
@@ -0,0 +1,13 @@
+public class Level0 {
+
+        public static void main(String[] args){
+
+            System.out.println("____________________________________________________________\n");
+            System.out.println(" Hello! I'm Duke\n");
+            System.out.println(" What can I do for you?\n");
+            System.out.println("____________________________________________________________\n");
```

Comment:
This few lines seem to be repeated quite a number of times. Perhaps you can abstract them out into a method? Also, since these strings are never changed during runtime, you can also define them as a constant.